src/models/nftRewardTier.ts

```diff
@@ -7,7 +7,8 @@ export type NftRewardTier = {
   remainingSupply: number | undefined
   imageUrl: string // link to ipfs
   name: string
-  tierRank: number // cheapest tier is 1
+  /* @deprecated - now derived from comparing contributionFloor between each */
```

Comment:
Should we create a TODO issue for this? Once we launch the new Create flow, we'll be able to delete this right?

Reply:
yessir